examples/plot_grad_search_cv.py

```diff
@@ -72,15 +73,19 @@
 ##############################################################################
 # Now do the hyperparameter optimization with implicit differentiation
 # --------------------------------------------------------------------
+
+clf = sklearn.linear_model.Lasso(
+    fit_intercept=False, max_iter=1000, warm_start=True, tol=tol)
+
 print('sparse-ho started')
 
 t0 = time.time()
 Model = Lasso
 Criterion = CV
 log_alpha0 = np.log(alpha_max / 10)
 monitor_grad = Monitor()
-criterion = CrossVal(X, y, Lasso, cv=kf)
-algo = ImplicitForward(criterion, use_sk=True, max_iter=max_iter)
+criterion = CrossVal(X, y, Lasso, cv=kf, clf=clf)
```

Comment:
clf stands for classifier. Call it estimator

Reply:
```suggestion
criterion = CrossVal(X, y, Model, cv=kf, clf=clf)
```
?

Reply:
done thx